pkg/cmd/adm/register_member_test.go

```diff
@@ -35,16 +35,19 @@ func TestRegisterMember(t *testing.T) {
 	hostKubeconfig := PersistKubeConfigFile(t, HostKubeConfig())
 	memberKubeconfig := PersistKubeConfigFile(t, MemberKubeConfig())
 
+	hostNs := "toolchain-host-operator"
+	memberNs := "toolchain-member-operator"
+
```

Comment:
all the occurrences could have been replaced with the constants from toolchain-common repo `test.HostOperatorNs` and `test.MemberOperatorNs` (it's used at many places already)